terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 1:  4944 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 2:  4946 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 3:  4948 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 4:  4950 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 5:  4952 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 6:  4954 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 7:  4956 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 8:  4958 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 9:  4960 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 10:  4962 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 11:  4964 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 12:  4966 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 13:  4968 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 14:  4970 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 15:  4972 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 16:  4974 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 17:  4976 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 18:  4978 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 19:  4980 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 20:  4982 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 21:  4984 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 22:  4986 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 23:  4988 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 24:  4990 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 25:  4992 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 26:  4994 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 27:  4996 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 28:  4998 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 29:  5000 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 30:  5002 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 31:  5004 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 32:  5006 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 33:  5008 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 34:  5010 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 35:  5012 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 36:  5014 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 37:  5016 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 38:  5018 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 39:  5020 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 40:  5022 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 41:  5024 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 42:  5026 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 43:  5028 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 44:  5030 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 45:  5032 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 46:  5034 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 47:  5036 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 48:  5038 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 49:  5040 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 50:  5042 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 51:  5044 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 52:  5046 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 53:  5048 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 54:  5050 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 55:  5052 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 56:  5054 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 57:  5056 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 58:  5058 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 59:  5060 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 60:  5062 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 61:  5064 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 62:  5066 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 63:  5068 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 64:  5070 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 65:  5072 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 66:  5074 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 67:  5076 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 68:  5078 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 69:  5080 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 70:  5082 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 71:  5084 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 72:  5086 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 73:  5088 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 74:  5090 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 75:  5092 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 76:  5094 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 77:  5096 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 78:  5098 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 79:  5100 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 80:  5102 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 81:  5104 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 82:  5106 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 83:  5108 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 84:  5110 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 85:  5112 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 86:  5114 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 87:  5116 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 88:  5118 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 89:  5120 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 90:  5122 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 91:  5124 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 92:  5126 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 93:  5128 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 94:  5130 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/20.neglite.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-3-2-c-val.sh: line 95:  5132 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-c.pddl $1
